TileSystem/TileSystem.csproj

```diff
@@ -59,14 +60,19 @@
     <Compile Include="Interfaces\Management\Args\TileRemovedArgs.cs" />
     <Compile Include="Interfaces\Management\IManageAreas.cs" />
     <Compile Include="Interfaces\Management\IManageEntities.cs" />
+    <Compile Include="Interfaces\Management\IManageTileChange.cs" />
     <Compile Include="Interfaces\Management\IManageTiles.cs" />
     <Compile Include="Interfaces\Solvers\ISolver.cs" />
     <Compile Include="Interfaces\TileChange\Args\TileChangedArgs.cs" />
     <Compile Include="Interfaces\TileChange\IChangeTile.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="Utils\TileChangeType.cs" />
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Folder Include="Implementation\2d\" />
```

Comment:
Damn didn't see this, will update to remove these folders from here so we don't get errors before I start doing this work.